### PR TITLE
Flash alerts

### PR DESCRIPTION
**Pull request by spokrovskiy**
added flash alerts partial and error partial

### CODE REVIEW

File: app/views/layouts/_errors.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<% errors.each do | message| %>
+  <div data-alert class="alert-box alert radius">
+    <%= message  %>
```
Comment (jraenorris) [inline app/views/layouts/_errors.html.erb]: There are two spaces after 'message' when there should be one

File: app/views/layouts/_flash.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<% flash.each do |type, message| %>
+  <div data-alert class="alert-box <%= type %> radius">
+    <%= message  %>
```
Comment (jraenorris) [inline app/views/layouts/_flash.html.erb]: here too

File: Gemfile.lock [hunk 3]
```diff
@@ -74,12 +74,12 @@ GEM
     factory_girl_rails (4.7.0)
       factory_girl (~> 4.7.0)
       railties (>= 3.0.0)
+    font-awesome-rails (4.6.3.1)
```
Comment (jraenorris) [inline Gemfile.lock]: Why is this changing here? It looks like one group might have a different Gemfile.lock than the other so each time one merges in, it means the other has a diff from master. Try to get everyone on your team using the same Gemfile.lock over the course of today. One suggestion of how to do this is by not committing changes to your Gemfile.lock on branches that do not need to actually change /update it

File: app/views/layouts/_errors.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<% errors.each do |git message| %>
```
Comment (jraenorris) [inline app/views/layouts/_errors.html.erb]: git message?

File: spec/features/venues/create_new_venue_spec.rb [hunk 5]
```diff
@@ -20,5 +20,7 @@
     expect(page).to have_content(venue['website'])
     expect(page).to have_content(venue['address'])
     expect(page).to have_content(t_is_accessible_string)
+
+    expect(page).to have_content('Venue saved successfully')
   end
```
Comment (jraenorris) [inline spec/features/venues/create_new_venue_spec.rb]: This is only testing the happy path. You should test that your flash messages are showing with an unhappy path test or two

Comment (nvanselow) [inline spec/features/venues/create_new_venue_spec.rb]: This code should not be part of this pull request. I think this is another one of those branching from a branch issues.

Comment (jraenorris) [inline spec/features/venues/create_new_venue_spec.rb]: Note: talked offline

File: spec/features/venues/create_new_venue_spec.rb [hunk 6]
```diff
@@ -20,5 +22,15 @@
...
     expect(page).to have_content(t_is_accessible_string)
+
+    expect(page).to have_content('Venue saved successfully')
+  end
+
+  scenario 'User submits blank form' do
+    click_button('Create Venue')
+
+    expect(page).to have_content('Problems saving venue')
+    expect(page).to have_content('Name can\'t be blank')
```
Comment (jraenorris) [inline spec/features/venues/create_new_venue_spec.rb]: Double quotes to avoid the `\'` would be more readable

File: spec/features/venues/create_new_venue_spec.rb [hunk 7]
```diff
@@ -20,5 +22,15 @@
...
+    expect(page).to have_content('Address can\'t be blank')
```
Comment (jraenorris) [inline spec/features/venues/create_new_venue_spec.rb]: ^
